htdocs/adherents/subscription.php

```diff
@@ -700,42 +679,29 @@
                 print '<td align="right">'.$langs->trans("Account").'</td>';
             }
             print "</tr>\n";
-
+            
+            $subscriptionstatic=new SubscriptionPlus($db);
```

Comment:
What is this ?

Reply:
@eldy bad paste from my alternative member module. I fixed it